data_management/views.py

```diff
@@ -194,7 +194,7 @@ def external_object(request, doi, title, version):
 
     # Find the external object
     try:
-        external_object = models.ExternalObject.objects.get(Q(doi_or_unique_name=doi) & Q(title=title) & Q(version=version))
+        external_object = models.ExternalObject.objects.get(Q(other_unique_name=doi) & Q(title=title) & Q(version=version))
```

Comment:
I'm not sure what this is doing here, but `doi` sounds like an identifier? Presumably you actually need to test equality for `other_unique_name` and `identifier` in fact?

Reply:
I renamed the variable `doi` in this function and in the urls.py file to `other_unique_name`, which makes more sense now (this will not be used at all if someone has specified an `identifier`).